packages/muelu/src/Graph/PairwiseAggregation/MueLu_NotayAggregationFactory_def.hpp

```diff
@@ -192,9 +192,7 @@ namespace MueLu {
     ordering = O_NATURAL;
     if (orderingStr == "random" ) ordering = O_RANDOM;
     else if(orderingStr == "natural") {}
-#if defined(HAVE_MUELU_KOKKOS_REFACTOR)
```

Comment:
The Cuthill-McKee algorithm is only implemented in Kokkos this option should not be exposed to non-kokkos path

Reply:
@lucbv The entire file is encapsulated in `#ifdef HAVE_MUELU_KOKKOS_REFACTOR` (cf. line 49), so this statement can safely be removed, as it just repeats the same `#ifdef`. Same argument holds for all other changes in this file.